File: api/src/main/java/br/cefet/aida/model/Torneio.java
package br.cefet.aida.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 *
 * @author dougl
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Torneio {
    private int idTorneio;
    private String nome;
    private boolean iniciado;
    private int quantidadeDeEquipes;
    private String regras;
    private int idUsuario;
    private int idModalidade;
    private Modalidade modalidade;
    private Usuario usuario;
}

File: site/src/app/model/torneio.ts
import { Modalidade } from "./modalidade";
import { Usuario } from "./usuario";

export class Torneio {
    idTorneio: number;
    nome: string;
    iniciado: boolean;
    quantidadeDeEquipes: number;
    regras: string;
    idUsuario: number;
    idModalidade: number;
    modalidade: Modalidade;
    usuario: Usuario;

    constructor() {
        this.idTorneio = 0;
        this.nome = "";
        this.iniciado = false;
        this.quantidadeDeEquipes = 0;
        this.regras = "";
        this.idUsuario = 0;
        this.idModalidade = 0;
        this.modalidade = new Modalidade();
        this.usuario = new Usuario();
    }
}
